libinitializer/P2PInitializer.cpp

```diff
@@ -167,3 +171,49 @@ void P2PInitializer::initConfig(boost::property_tree::ptree const& _pt)
         exit(1);
     }
 }
+
+void P2PInitializer::resetWhitelist(const std::string& _configFile)
+{
+    boost::property_tree::ptree pt;
+    boost::property_tree::read_ini(_configFile, pt);
+
+    auto whitelist = parseWhitelistFromPt(pt);
+
+    p2pService()->setWhitelist(whitelist);
+}
+
+PeerWhitelist::Ptr P2PInitializer::parseWhitelistFromPt(boost::property_tree::ptree const& _pt)
```

Comment:
Do not use ugly abbreviation, like pt as property_tree

Reply:
done